core/src/main/java/org/radargun/stages/JMXInvocationStage.java

```diff
@@ -107,6 +113,11 @@ public DistStageAck executeOnSlave() {
             } else if (operationType == OperationType.GET_ATTRIBUTE_VALUE) {
                log.trace("Getting value of attribute " + targetName);
                result = connection.getAttribute(objectInstance.getObjectName(), targetName);
+            } else if (operationType == OperationType.SET_ATTRIBUTE_VALUE) {
+               log.trace("Setting value of attribute " + targetName + " to " + methodParameters.get(0).getElementValue().toString());
```

Comment:
I believe this will end up with NPE when 'methodParameters' is not specified. Could you please add a check for the list size (!= 0) when setting attribute value?